bait-files/src/main/webapp/review.js

```diff
@@ -44,28 +45,43 @@ function getRequest() {
   fetch('/request' + queryString)
       .then(response => response.json())
       .then((request) => {
-          addRequestToPage(request);
+        addRequestToPage(request);
+        addImageToPage(request);
       })
       .then(getPanels);
 }
 
-
 function addRequestToPage(request) {
-    addTextToPage(REQUEST_ID_CONTAINER, request.requestId);
-    addTextToPage(USER_ID_CONTAINER, request.userId);
-    addTextToPage(STATUS_CONTAINER, request.status);
-    addTextToPage(DATE_CONTAINER, new Date(request.submissionDate).toLocaleDateString() + '  ' +
-                new Date(request.submissionDate).toLocaleTimeString());
-    addTextToPage(PHONE_INPUT, request.phoneNum);
-    addTextToPage(NAME_INPUT, request.name);
-    addTextToPage(USERNAME_INPUT, request.username);
-    addTextToPage(EMAIL_INPUT, request.email);
-    addTextToPage(ADDRESS_INPUT, request.address);
-    addTextToPage(NOTES_INPUT, request.notes);
-    document.getElementById(REVIEW_REQUEST_ID).value = request.requestId;
-    document.getElementById(REVIEW_USER_ID).value = request.userId;
+  addTextToPage(REQUEST_ID_CONTAINER, request.requestId);
+  addTextToPage(USER_ID_CONTAINER, request.userId);
+  addTextToPage(STATUS_CONTAINER, request.status);
+  addTextToPage(
+      DATE_CONTAINER,
+      new Date(request.submissionDate).toLocaleDateString() + '  ' +
+          new Date(request.submissionDate).toLocaleTimeString());
+  addTextToPage(PHONE_INPUT, request.phoneNum);
+  addTextToPage(NAME_INPUT, request.name);
+  addTextToPage(USERNAME_INPUT, request.username);
+  addTextToPage(EMAIL_INPUT, request.email);
+  addTextToPage(ADDRESS_INPUT, request.address);
+  addTextToPage(NOTES_INPUT, request.notes);
+  document.getElementById(REVIEW_REQUEST_ID).value = request.requestId;
+  document.getElementById(REVIEW_USER_ID).value = request.userId;
 }
 
 function addTextToPage(containerId, text) {
-    document.getElementById(containerId).appendChild(document.createTextNode(text));
-}
\ No newline at end of file
+  document.getElementById(containerId)
+      .appendChild(document.createTextNode(text));
+}
+
+function addImageToPage(request) {
+  let blobKeyString = request.image;
+  if (blobKeyString != 'noKey') {
```

Comment:
Didn't we say that we should check for undefined here instead of 'noKey'?

Reply:
hm, it seems that this is an older version and my pushes since haven't been reflected on here yet